backend/tracim_backend/config.py

```diff
@@ -173,7 +187,7 @@ def __init__(self, settings: typing.Dict[str, typing.Any]):
         self._check_consistency()
         self.check_config_validity()
         logger.debug(self, "CONFIG_PROCESS:5: End of config process")
-        self.IDP_LIST: typing.List[str] = []
+        self.IDP_LIST: typing.List[IdPConfig] = []
```

Comment:
I suggest to name this with a more explicit name; something like SAML_IDP_LIST? (is this related to SAML or to SHIBB?)

Reply:
done